NEWS.md

```diff
@@ -1,5 +1,6 @@
 ## 2020-06-05 v2.1.1-SNAPSHOT
 * [MODDICORE-62](https://issues.folio.org/browse/MODDICORE-62) Adjusted handling of repeatable fields
+* [MODDICORE-52](https://issues.folio.org/browse/MODDICORE-52) Add support for MappingEngine work with a leader
```

Comment:
do we plan a bugfix for this? if so, then it has to be applied on release branch as well, where version is 2.1.2. Otherwise, it should be included in the next iteration 2.2.0-SNAPSHOT